Core/tooling/build.sh

```diff
@@ -2,15 +2,50 @@
 
 set -euxo pipefail
 
+# fix openpgp in node_modules
+echo "Patching openpgp.js v5..."
+for f in openpgp.min.js openpgp.min.js.map openpgp.min.mjs openpgp.min.mjs.map openpgp.mjs;
+do
+  if [ -f node_modules/openpgp/dist/$f ]; then rm -f node_modules/openpgp/dist/$f ; fi
+  if [ -f node_modules/openpgp/dist/node/$f ]; then rm -f node_modules/openpgp/dist/node/$f ; fi
+done
+extra_exports="
+// -----BEGIN ADDED BY FLOWCRYPT----
+exports.readToEnd = readToEnd;
```

Comment:
This solution works but is too hacky if the other alternative is to add type definitions as suggested in https://github.com/openpgpjs/openpgpjs/discussions/1478#discussioncomment-2149306 and https://github.com/openpgpjs/web-stream-tools/issues/10

You don't initially even need to make any PR there. You can simply `import readToEnd as untypedReadToEnd fron 'openpgp/web-stream-tools'` and then `const readToEnd = untypedReadToEnd as WhatEverTypeYouWant`

That is much cleaner than approach here.

Reply:
I'm not a big specialist in writing typescript definitions, so I unlikely will do it, and they unlikely will do in the neartest time. So I see this as easiest and the most efficient way. This way we don't need to deal with `web-stream-tools` separately. I assume they are just included into final JavaScript source of the library and they don't plan to change this, although they don't export that explicitly.